MEME/MEME/Model/ModelHomeView/View/ModelHomeViewController.swift

```diff
@@ -0,0 +1,403 @@
+        textField.attributedPlaceholder = NSAttributedString(string: "원하는 메이크업을 검색해보세요.", attributes: placeholderAttributes)
+        searchBar.tintColor = .mainBold
+        
+        return searchBar
+    }()
+    
+    private var modelWelcomeLabel: UILabel = {
+        let label = UILabel()
+        label.text = "환영합니다!"
+        label.textColor = .black
+        label.font = .pretendard(to: .semiBold, size: 20)
+        label.numberOfLines = 0
+        
+        return label
+    }()
+    private var modelWelcomeGuideLabel: UILabel = {
+        let label = UILabel()
+        label.text = "아티스트와의 약속 놓치지 마세요!"
+        label.textColor = .black
+        label.font = .pretendard(to: .semiBold, size: 20)
+        
+        return label
+    }()
+    private lazy var viewAllReservationsButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("전체 예약 보기", for: .normal)
+        button.setTitleColor(.black, for: .normal)
+        if let image = UIImage(systemName: "chevron.right")?.withTintColor(.black, renderingMode: .alwaysOriginal) {
+            button.setImage(image, for: .normal)
+        }
+        button.semanticContentAttribute = .forceRightToLeft
+        button.addTarget(self, action: #selector(viewAllReservationsTapped), for: .touchUpInside)
+        
+        return button
+    }()
+    private let collectionView = UICollectionView(frame: .zero, collectionViewLayout: .init())
+    
+    // MARK: - Property
+    private let disposeBag = DisposeBag()
+    private var viewModel = ModelHomeViewModel()
+    private var dataSource: UICollectionViewDiffableDataSource<ModelHomeViewControllerSection, ModelHomeViewControllerItem>?
+    
+    //MARK: - Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        
+        setUserNickName()
+        setupCollectionView()
+        addSubViews()
+        makeConstraints()
+        setupSearchBar()
+        bindViewModel()
+    }
+    
+    // MARK: - addSubviews
+    func addSubViews() {
+        contentsView.addSubview(memeLogoImageView)
+        contentsView.addSubview(alarmImageView)
+        contentsView.addSubview(searchMakeup)
+        contentsView.addSubview(modelWelcomeLabel)
+        contentsView.addSubview(modelWelcomeGuideLabel)
+        contentsView.addSubview(viewAllReservationsButton)
+        contentsView.addSubview(collectionView)
+        scrollView.addSubview(contentsView)
+        view.addSubview(scrollView)
+    }
+    
+    // MARK: - makeConstraints
+    func makeConstraints() {
+        scrollView.snp.makeConstraints { (make) in
+            make.edges.equalTo(view.safeAreaLayoutGuide)
+        }
+        contentsView.snp.makeConstraints { (make) in
+            make.edges.equalTo(scrollView)
+            make.width.equalTo(scrollView)
+        }
+        memeLogoImageView.snp.makeConstraints {make in
```

Comment:
```suggestion
        memeLogoImageView.snp.makeConstraints { make in
```

중괄호 뒤에 띄어쓰기 한 칸 부탁드립니다!